src/renderer/containers/ReplayBrowser/ReplayFile.tsx

```diff
@@ -28,6 +28,8 @@ export interface ReplayFileProps extends FileResult {
   onSelect: () => void;
   onPlay: () => void;
   onOpenMenu: (index: number, element: HTMLElement) => void;
+  handleAddToList: (name: string) => void;
+  list: Array<string>;
```

Comment:
I don't want `ReplayFile` to have to care about the list of items. I would rather it simply have a `selected` prop and an `onClick` prop that it gets passed from the parent.

```suggestion
  onClick: () => void;
  selected: boolean;
```

The basic styles you added should then only be shown if `selected` is true.